package.json

```diff
@@ -38,18 +43,22 @@
     "jquery-validation": "^1.19.1",
     "lodash.attempt": "^4.2.0",
     "lodash.debounce": "^4.0.8",
+    "lozad": "^1.14.0",
     "medium-zoom": "^1.0.4",
     "moment": "^2.23.0",
     "moment-timezone": "^0.5.23",
     "mustache": "^3.1.0",
     "noty": "^3.2.0-beta",
+    "perfect-scrollbar": "^1.4.0",
     "popper.js": "^1.16.0",
     "postcss-flexbugs-fixes": "^4.1.0",
     "postcss-preset-env": "^6.6.0",
     "rails-erb-loader": "^5.5.2",
     "select2": "^4.0.6-rc.1",
     "select2-bootstrap-theme": "^0.1.0-beta.10",
     "slick-carousel": "^1.8.1",
+    "sortablejs": "^1.10.1",
+    "stacked-menu": "^1.1.12",
```

Comment:
Why did we need to add this based on my work to add this in?

Reply:
you can remove